modules/auth_radius/authrad_mod.c

```diff
@@ -90,11 +91,12 @@ static cmd_export_t cmds[] = {
  * Exported parameters
  */
 static param_export_t params[] = {
-	{"radius_config",    PARAM_STRING, &radius_config	},
```

Comment:
Reverted this but the alignment is out-of-wack and there's a mixture of tabs and spaces